sklearn/metrics/classification.py

```diff
@@ -313,8 +315,11 @@ class labels [2].
     .. [1] J. Cohen (1960). "A coefficient of agreement for nominal scales".
```

Comment:
I'm not sure whether we prefer DOIs or URLs, or both... check the rest of the codebase.